### PR TITLE
- Config Changes for Logo and Jwt Expiry

### PR DESCRIPTION
**Pull request by mohityadav766**
### Describe your changes :

- Added Logo Configuration (openmetadata, with Url , with Filepath)
- Updated JwtToken expiry time to be configurable

#
### Type of change :

- [x] Improvement

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#

### CODE REVIEW

**Comment by mohityadav766** [comment]
@Sachin-chaurasiya this is the PR for logo changes you needed

**Comment by Sachin-chaurasiya** [comment]
> @Sachin-chaurasiya this is the PR for the logo changes you needed,
> 
> api/v1/system/config/applicationConfig -> url will give the following config
> 
> {
>     "logoConfig": {
>         "logoLocationType": "openmetadata",
>         "logoAbsoluteFilePath": "",
>         "logoUrlPath": ""
>     },
>     "slackConfig": {
>         "slackUrl": "https://slack.open-metadata.org/"
>     }
> }
> 
> -         "logoLocationType": "openmetadata",  is the default one
> -         "logoLocationType": "url", -> when we get image url for logo, value will be in "logoUrlPath"
> -         "logoLocationType": "filePath", -> when we get file path for logo, value will be in "logoAbsoluteFilePath"
> 
> 
> Let me know if you need any help with this

Thanks @mohityadav766 , will have a look into it 👌

**Comment by Sachin-chaurasiya** [comment]
> @Sachin-chaurasiya this is the PR for the logo changes you needed,
> 
> api/v1/system/config/applicationConfig -> url will give the following config
> 
> {
>     "logoConfig": {
>         "logoLocationType": "openmetadata",
>         "logoAbsoluteFilePath": "",
>         "logoUrlPath": ""
>     },
>     "slackConfig": {
>         "slackUrl": "https://slack.open-metadata.org/"
>     }
> }
> 
> -         "logoLocationType": "openmetadata",  is the default one
> -         "logoLocationType": "url", -> when we get image url for logo, value will be in "logoUrlPath"
> -         "logoLocationType": "filePath", -> when we get file path for logo, value will be in "logoAbsoluteFilePath"
> 
> 
> Let me know if you need any help with this

Thanks @mohityadav766 , will have a look into it 👌

File: conf/openmetadata.yaml [hunk 1]
```diff
@@ -266,9 +266,16 @@ email:
   transportationStrategy: ${SMTP_SERVER_STRATEGY:-"SMTP_TLS"}
 
 sandboxModeEnabled: ${SANDBOX_MODE_ENABLED:-false}
```
Comment (harshach) [inline conf/openmetadata.yaml:268]: this can be removed as well cc @chirag-madlani

**Comment by harshach** [comment]
@chirag-madlani we should be able to replace the logo 
image
image
These are different size logos right. We should publicize what size of logos we need and probably would need different assets.

Also at the footer we should be calling out "Powered By OpenMetadata"

**Comment by chirag-madlani** [comment]
@harshach Will share the size for logs, Update the footer as well.
cc: @devyani-kaushik @Sachin-chaurasiya

File: conf/openmetadata.yaml [hunk 2]
```diff
@@ -266,9 +266,16 @@ email:
...
-slackChat:
-  slackUrl: ${SLACK_CHAT_SLACK_URL:-"https://slack.open-metadata.org/"}
+
+applicationConfig:
+  logoConfig:
+    logoLocationType: ${OM_LOGO_LOCATION_TYPE:-openmetadata} #either "openmetadata' or { "url" or "filePath" , base on this specify either 'logoAbsoluteFilePath' or 'logoUrlPath' }
+    logoAbsoluteFilePath: ${OM_LOGO_LOCATION_FILE_PATH:-""}
+    logoUrlPath: ${OM_LOGO_LOCATION_URL_PATH:-""}
+  slackConfig:
```
[the remark this answers is not in the document]
Comment (mohityadav766) [reply conf/openmetadata.yaml]: done

**Comment by mohityadav766** [comment]
@chirag-madlani we should take the Jwt Token expiry time from the the above endpoint as well , and use that in case to deduce the refresh token timing , it will give the time in seconds

**Comment by mohityadav766** [comment]
@chirag-madlani @Sachin-chaurasiya can you please review the UI changes?!

File: conf/openmetadata.yaml [hunk 3]
```diff
@@ -265,10 +265,15 @@ email:
...
+applicationConfig:
+  logoConfig:
+    logoLocationType: ${OM_LOGO_LOCATION_TYPE:-openmetadata} #either "openmetadata' or { "url" or "filePath" , based on this specify either '*AbsoluteFilePath' or '*LogoUrlPath' }
+    loginPageLogoAbsoluteFilePath: ${OM_LOGO_LOGIN_LOCATION_FILE_PATH:-""} #login page logo , work in "filePath" mode
+    loginPageLogoUrlPath: ${OM_LOGO_LOGIN_LOCATION_URL_PATH:-""} #login page logo , work in "url" mode
+    navBarLogoAbsoluteFilePath: ${OM_LOGO_NAVBAR_LOCATION_FILE_PATH:-""} #nav bar logo , work in "filePath" mode
+    navBarLogoUrlPath: ${OM_LOGO_NAVBAR_LOCATION_URL_PATH:-""} #nav bar logo , work in "url" mode
+  loginConfig:
+    maxLoginFailAttempts: ${OM_MAX_FAILED_LOGIN_ATTEMPTS:-3}
+    accessBlockTime: ${OM_LOGIN_ACCESS_BLOCKTIME:-600}
```
Comment (harshach) [inline conf/openmetadata.yaml:277]: what is this config do?

**Review by harshach — APPROVED** [review]

File: conf/openmetadata.yaml (hunk 3 above)
Comment (mohityadav766) [reply conf/openmetadata.yaml:277]: maxLoginIsAttemts -> Maximum allowed login attempts before blocking login for "accessBlockTime"
jwtExpiryTime -> control the time of expiry for the login jwt token

**Pull request merged**